tools/build_h3_tools.sh

```diff
@@ -186,8 +186,9 @@ LDFLAGS=${LDFLAGS:-"-Wl,-rpath,${OPENSSL_LIB}"}
 
 # Then nghttp3
 echo "Building nghttp3..."
-[ ! -d nghttp3 ] && git clone --depth 1 -b v1.1.0 https://github.com/ngtcp2/nghttp3.git
+[ ! -d nghttp3 ] && git clone --depth 1 -b v1.2.0 https://github.com/ngtcp2/nghttp3.git
```

Comment:
Thanks for working on keeping this updated. 

I have generally kept this build_h3_tools.sh script on parity with the curl one:
https://curl.se/docs/http3.html

Since that is still sticking with nghttp3 1.1.0, I'm a bit biased to keep it that way. Are there changes in nghttp3 that we want?

To be clear: I'm not opposed to this change necessarily, I just want to make sure I understand the motivation for updating beyond curl's versions.

Reply:
Not really, the important change we need is the one from nghttp2 `1.60.0`  which fixes few things like the `h2load`.